### ISSUE TITLE
Shift + return = submit

### ISSUE THREAD

**Issue by jeffdomke** [blockstack-radiks/banter#54]
Clicking the little button is pretty annoying. Think this could be subtle tool tip next to the button

**Comment by aulneau** [comment]
shift return typically means inline breaks. 

Twitter behaves this way (needing to hit the button)

**Comment by jeffdomke** [comment]
Right, right. Could be another key. I can't really argue with the twitter example. 😔
